frontend/src/components/Pagination.js

```diff
@@ -30,9 +30,9 @@ const Pagination = (props) => {
         )
       )}
       {props.currentPageIndex !== props.numberOfPages - 1 &&
-        renderPaginationBox("Next", props.currentPageIndex + 1)}
+        renderPaginationBox("Back", props.currentPageIndex + 1)}
```

Comment:
Lets pull the changes in Pagination out, as someone else has been working on it.  Everything else looks solid!